modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -230,6 +232,60 @@ const FrontendDataSet = ({
 		setItems(dataSetData.items);
 	}
 
+	useEffect(() => {
+		loadClientExtensions([
+			{
+				handler: (filterModules) => {
```

Comment:
:sweat_smile:  Great! Many thanks!!!

I was looking for a better word starting with a letter that followed `i` so that I could put the handler below the `imports` but couldn't find any :joy: